cypress/component/admin/Popup.cy.jsx

```diff
@@ -0,0 +1,99 @@
+import Popup from "@/components/admin/Popup";
+import React from "react";
+// import { useState } from "react";
+
+describe("Popup Component", () => {
+  it("opens and displays the popup", () => {
+    const popup = {
+      title: "Sample Popup",
+      text: "This is a test popup.",
+      color: "green",
+      visible: true,
+    };
+
+    const onClick = cy.stub();
```

Comment:
Take a look at the tests that have a parent component and refer to the hooks being used